tests/aks_test.go

```diff
@@ -0,0 +1,101 @@
+package aks_test
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	v1 "k8s.io/api/apps/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/azure"
+)
+
+var _ = Describe("AKS-Pipeline Tests : ", func() {
```

Comment:
It would apply to all tests in this file/ defined within this method.  How about AKS-Deployment ( i meant K8s Deployments :) )   tests ? can't think of better one either

Reply:
sure